### DISCUSSION TITLE
Project rename + snap sandbox support ?

### DISCUSSION THREAD

**Discussion by Joffreybvn** [Eloston/ungoogled-chromium#1506] [Q&A]
Hello @Eloston, hello everyone,

A month ago, I created a [snap package](https://snapcraft.io/ungoogled-chromium) of the Ubuntu Focal version. This way, today, 1500 others peoples and me are able to install Ungoogled Chromium easily on many different linux distributions. 

But there's a problem: The Chromium [sandbox can not be used within the snap](https://github.com/Joffreybvn/ungoogled-chromium-snapcraft/issues/2#issuecomment-828537341). Canonical restrict the use of navigator's sandbox to trusted publishers only. To be a trusted publisher, this project needs:
- Another name. "Uncloudium" is a good name !
- Be published by the creators.

### This discussion is about making this snap and Ungoogled Chromium a bit more "legal":
 - Are you still planning to rename your fork ? (I can help with that)

- Would it be possible to move [my repo](https://github.com/Joffreybvn/ungoogled-chromium-snapcraft) under your organization ?

**Comment by Eloston** [comment]
> Are you still planning to rename your fork ? (I can help with that)

No plans, no good reason to

> Would it be possible to move my repo under your organization ?

I took a quick look. Maybe it'd be better to use Portable Linux since it's more distro-agnostic?

**Reply by Joffreybvn** [reply]
Yes, I'm not happy with the Focal version: the snap image is very heavy and doesn't display correctly on older linux versions. I'll build the snap for Chromium 90 with portable linux.

About the rename, until this project becomes trusted on Snapcraft.io, I will publish two versions:
- One for Snap store, without the chromium sandbox.
- One on my repo, with the chromium sandbox.
